flow/FastAlloc.cpp

```diff
@@ -304,7 +304,7 @@ void *FastAllocator<Size>::allocate() {
 
 #if VALGRIND
 	if (valgrindPrecise()) {
-		return malloc(Size);
+		return aligned_alloc(Size, Size);
```

Comment:
Is this to match the behavior of the normal fast allocator, or does it improve the valgrind results somehow? I can't remember what the alignment story is with fast allocator now.

Reply:
FastAllocator<4096> not aligning to 4096 was causing [this assert](https://github.com/apple/foundationdb/blob/master/fdbrpc/AsyncFileKAIO.actor.h#L622) (or a similar one I can't remember) to fail. This is to match the behavior of the normal fast allocator